rsconnect/actions.py

```diff
@@ -377,7 +378,8 @@ def deploy_python_api(connect_server, directory, extra_files, excludes, entry_po
     :return: the ultimate URL where the deployed app may be accessed and the sequence
     of log lines.  The log lines value will be None if a log callback was provided.
     """
-    app_store = AppStore(directory)
+    module_file = join(directory, entry_point.split[':'][0] + '.py')
```

Comment:
Should we be degrading more gracefully if `entry_point` has no `:`? Or are we assuming this is always called after `_validate_entry_point`?

Reply:
I thought about this for a while.  For now, I've landed on the side of the latter.  I do think though that we should do a follow-on change where all the input validation moves to the `actions` layer out of `main`.